tests/integration/frameworks/test_pytorch_lightning_impl.py

```diff
@@ -1,15 +1,18 @@
 import math
 
-import torch
 import pandas as pd
 import psutil
 import pytest
 import torch.nn as nn
-import pytorch_lightning as pl
+import setuptools  # type: ignore[reportUnusedImport] noqa: F401
 
 import bentoml
 from tests.utils.helpers import assert_have_file_extension
 
+import torch  # isort:skip
```

Comment:
?

Reply:
try to fix this https://github.com/pypa/setuptools/issues/2230 but not work. will ignore it